crypto/crypto_int_test.go

```diff
@@ -84,19 +85,6 @@ func decodeHex(s string) []byte {
 	return d
 }
 
-// returns true if function called panic
-func shouldPanic(f func()) (didPanic bool) {
```

Comment:
Not sure if this is actually dead code, but unless there is some fancy introspection test magic happening, I don't think this is being called anywhere?